src/main/java/org/sagebionetworks/bridge/exporter/dynamo/DynamoHelper.java

```diff
@@ -95,18 +95,31 @@ public StudyInfo getStudyInfo(String studyId) {
         // it's not null. (This should cover both cases where the attribute doesn't exist and cases where the attribute
         // exists and is null.
         StudyInfo.Builder studyInfoBuilder = new StudyInfo.Builder();
-        if (studyItem.get(STUDY_INFO_KEY_DATA_ACCESS_TEAM) != null) {
-            studyInfoBuilder.withDataAccessTeamId(studyItem.getLong(STUDY_INFO_KEY_DATA_ACCESS_TEAM));
-        }
 
-        if (studyItem.get(STUDY_INFO_KEY_PROJECT_ID) != null) {
-            studyInfoBuilder.withSynapseProjectId(studyItem.getString(STUDY_INFO_KEY_PROJECT_ID));
+        // we need to check at first if the study is disabled exporting or not
+        // treat null as false
+        boolean disableExport = false;
+
+        if (studyItem.get(STUDY_DISABLE_EXPORT) != null) {
+            if (studyItem.getInt(STUDY_DISABLE_EXPORT) != 0) {
+                disableExport = true;
+            }
```

Comment:
It might be worth writing a helper function or a utility to parse the DDB attribute as a boolean. This could be used for both disableExport and for usesCustomExportSchedule